src/main/java/edu/students/settler/service/impl/DormitoryServiceImpl.java

```diff
@@ -19,4 +22,9 @@
     public List<Dormitory> findAll() {
         return dormitoryRepository.findAll();
     }
+
+    @Override
+    public Set<Room> findDormitoryRooms(long dormitoryId) {
+        return dormitoryRepository.findById(dormitoryId).map(Dormitory::getRooms).orElse(Collections.emptySet());
```

Comment:
Not critical.

I suggest formatting code when you using `Optional`:
```java
return dormitoryRepository.findById(dormitoryId)
               .map(Dormitory::getRooms)
               .orElse(emptySet());
```
Also, use `static import` for `Collections` methods.

Reply:
matter of taste :)

Reply:
i understand that style i also was writing in that style when coding on java, but after i rethinked this style and now doing it oneline :)